.workflow-version

```diff
@@ -0,0 +1 @@
+$workflow_version='1.0.0.alpha1'
```

Comment:
I thought we were going with the name “beta”.

Reply:
We will be bumping to `beta` shortly after this release (timeline tbd).